plugins/corsair/fu-corsair-device.c

```diff
@@ -490,8 +417,6 @@ fu_corsair_device_to_string(FuDevice *device, guint idt, GString *str)
 				   idt,
 				   "DeviceKind",
 				   fu_corsair_device_type_to_string(self->device_kind));
-	fu_common_string_append_kx(str, idt, "InEndpoint", self->epin);
-	fu_common_string_append_kx(str, idt, "OutEndpoint", self->epout);
```

Comment:
I think you still need to print this, i.e. define a `->to_string()` vfunc on the bp device, and then call `fu_device_add_string(bp)` here to add all the BP properties.

Reply:
done